doc/make.jl

```diff
@@ -1,10 +1,8 @@
 # Install dependencies needed to build the documentation.
-ENV["JULIA_PKGDIR"] = joinpath(@__DIR__, "deps")
-using Pkg
-Pkg.init()
-cp(joinpath(@__DIR__, "REQUIRE"), Pkg.dir("REQUIRE"); force = true)
-Pkg.update()
-Pkg.resolve()
+using Pkg3
+empty!(DEPOT_PATH)
+pushfirst!(DEPOT_PATH, joinpath(@__DIR__, "deps"))
+pkg"up --manifest --fixed"
```

Comment:
Currently, the REPL API is a bit nicer than the "programmatic one" so I used it here by using the Pkg3 `pkg` string macro.

Reply:
If this solves the issue in https://github.com/JuliaLang/julia/pull/25036#issuecomment-387567446, I'll close that in lieu of this, which I see as superseding it.

Reply:
Since this changes the `DEPOT_PATH` it should also change the place where the precompile files are stored.